testing/sdk_security.py

```diff
@@ -215,64 +219,79 @@ def setup_security(framework_name: str,
 
     log.info("Setting up strict-mode security")
```

Comment:
Out of scope for this PR, but we could split this into a separate function as we are now only setting the permissions.